daemon/entrypoint.sh

```diff
@@ -226,6 +227,9 @@ function start_osd {
       activate)
          osd_activate
          ;;
+      devices)
```

Comment:
Does this really need another top-level OSD type?  Can this not be merged in with the existing `ceph-disk` stype OSD?  If it mandates compatibility breakage, then that is sufficient reason, of course.